spec/rage/sync_fiber_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rage/sync_fiber"
```

Comment:
Same issue here - such way this file will be applied to all tests, even the ones where we test the actual scheduler, e.g. [spec/fiber_spec.rb](https://github.com/rage-rb/rage/blob/master/spec/fiber_spec.rb).